crates/api/src/v1/package.rs

```diff
@@ -301,6 +310,16 @@ impl<'de> Deserialize<'de> for PackageError {
             },
             RawError::Conflict { status: _, ty, id } => match ty {
                 EntityType::NamespaceImport => Ok(Self::NamespaceImported(id.into_owned())),
+                EntityType::Record => Ok(Self::ConflictPendingPublish(
+                    id.parse::<AnyHash>()
```

Comment:
I personally find this easier to read but :shrug: 
```suggestion
                    AnyHash::from_str(&id)
```

Reply:
Fixed. Revised other occurrences as well.